.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Stale PR tracking
+
+on:
+  schedule:
+    - cron: '0 9 * * 1-5' # 9am on weekdays
```

Comment:
Do we want 9am UTC or something more US centric?

I'm fine with whatever time you decide, but let's say "9am UTC" in the comment so that it's clear that someone thought about the time zone.